implementations/rust/ockam/ockam_command/src/project/auth.rs

```diff
@@ -42,6 +42,21 @@ pub struct AuthCommand {
     trust_opts: TrustContextOpts,
 }
 
+fn parse_enroll_ticket(input: &str) -> Result<EnrollmentTicket> {
+    let ticket: EnrollmentTicket = match std::fs::read_to_string(input) {
```

Comment:
Is it ever worth it to be any more conservative with something like `std::path::Path::exists` to see if `input` is path-like before attempting to read from disk? If this idiom is already present elsewhere, that's fine, just a point of curiosity on my part. I guess that error path doesn't meaningfully change...

Reply:
The thing with `Path::exists`, is now it takes two attempts to check the path and then read it if the input is a path. I also believe most use cases will be a path instead of a plain text string.

This is why I chose to continue with the `read_to_string`